app/models/page_view.rb

```diff
@@ -38,6 +38,9 @@ class PageView < ApplicationRecord
 
     attributesForFaceting ["filterOnly(viewable_by)"]
 
+    attributeForDistinct :article_path
+    distinct true
```

Comment:
Would `visited_at_timestamp` be updated every time a new `page_view` is saved?

Reply:
`visited_at_timestamp` is the integer timestamp of the creation date of the page view